llvm/utils/TableGen/RegisterBankEmitter.cpp

```diff
@@ -112,7 +112,11 @@ class RegisterBankEmitter {
   void emitBaseClassDefinition(raw_ostream &OS, const StringRef TargetName,
                                const std::vector<RegisterBank> &Banks);
   void emitBaseClassImplementation(raw_ostream &OS, const StringRef TargetName,
-                                   std::vector<RegisterBank> &Banks);
+                                   const std::vector<RegisterBank> &Banks);
+  void emitRBIHeader(raw_ostream &OS, const StringRef TargetName,
+                     const std::vector<RegisterBank> &Banks);
+  void emitRBIIMPL(raw_ostream &OS, const StringRef TargetName,
```

Comment:
Suggest `emitRBIImplementation`

Reply:
I made this change. It shows up in the #71357

https://github.com/llvm/llvm-project/pull/71357